src/js/jsx/PanelSet.jsx

```diff
@@ -281,6 +291,14 @@ define(function (require, exports, module) {
                     <div ref="panelSet"></div>
                 );
             }
+        },
+        /**
+         * Close Design Space
+         *
+         * @private
+         */
+        _handleBackToPSClick: function () {
+            this.getFlux().actions.menu.native({ commandID: 5999 });
```

Comment:
Note to self: add a higher-level action for this. Note to @placegraphichere: ignore this note to self.